src/modos_schema/schema/modos_schema.yaml

```diff
@@ -283,3 +288,25 @@ enums:
       BCF:
         description: Binary call format, for efficient storage of sequence variation.
         meaning: EDAM:format_3020
+  CellType:
+    description: |
+      Cell type code from the cell ontology as a URI.
+      See: [https://bioregistry.io/registry/cl](https://bioregistry.io/registry/cl)
+    reachable_from:
+      source_ontology: bioregistry:cl
+      source_nodes:
+        - CL:0000000
+      include_self: false
+      relationship_types:
+        - rdfs:subClassOf
+  SourceMaterial:
+    description: |
+      Tissue or organ code from the UBERON ontology as a URI.
+      See: [https://bioregistry.io/registry/uberon](https://bioregistry.io/registry/uberon)
+    reachable_from:
+      source_ontology: bioregistry:uberon
+      source_nodes:
+        - UBERON:0001062
+      include_self: false
+      relationship_types:
+        - rdfs:subClassOf
```

Comment:
I don't see anything wrong here - but would play around with the prefix just to make sure that's not what's causing the faulty SHACL shape up above. Also not an expert in this yaml style of restricting stuff, we could look at it together if you like.

Reply:
yeah after our discussion you can disregard this idea ahaha

Reply:
Thanks for your input! Seeing all the issues with generated shacl, I'll just make everything string or uri instead of dynamic enums